algorithms/maths/extended_gcd.py

```diff
@@ -1,19 +1,15 @@
-def extended_gcd(a, b):
-    """Extended GCD algorithm.
-    Return s, t, g
-    such that a * s + b * t = GCD(a, b)
-    and s and t are co-prime.
-    """
+# Python program to demonstrate working of extended Euclidean Algorithm 
+	
+# function for extended Euclidean Algorithm 
+def gcdExtended(a, b): 
```

Comment:
use camel case. please reuse the existing method name `extended_gcd` , as it is failing the tests.
Please verify if unit tests are passing before submitting PR